src/components/common/MapDateSelect/DateSlider.tsx

```diff
@@ -3,12 +3,12 @@ import { useMemo } from 'react';
 
 type DateSliderProps = {
   dates: string[]; // date strings in the format 'YYYY-MM-DD'
```

Comment:
did we hear back on the PRD question regarding observation dates with similar month/year?

Reply:
What I was told is that it won't happen in practice, though for test data it's entirely possible (and will break the slider, as I see in the staging dev environment)

Reply:
Got it, the only thing off with test data is having multiple marks with same label. Not sure if that will be an issue. We'd have to use another test org (when we get to QA) with valid observation start dates on planting sites (not like the one I created in the dev org).